client/src/main/java/org/apache/rocketmq/client/impl/producer/DefaultMQProducerImpl.java

```diff
@@ -1326,12 +1326,10 @@ public TransactionSendResult sendMessageInTransaction(final Message msg,
                     }
 
                     if (localTransactionState != LocalTransactionState.COMMIT_MESSAGE) {
-                        log.info("executeLocalTransactionBranch return {}", localTransactionState);
-                        log.info(msg.toString());
+                        log.info("executeLocalTransactionBranch return: {} message: {}", localTransactionState, msg.toString());
```

Comment:
Remove the redundant `toString()` here.

Reply:
@aaron-ai thx for pointing it out,
done